src/webfaf2/templates/base.html

```diff
@@ -91,5 +91,15 @@
           </a>
         </div>
       </div>
+      {% if config['FEDMENU_URL'] is defined %}
```

Comment:
I'd prefer `{% if config['FEDMENU_URL'] and config['FEDMENU_DATA_URL'] %}` here so that `False` or `""` doesn't enable the block.
Or is there a reason for checking strictly for definition that I'm missing?